src/routes/[network]/(account)/fund/+page.svelte

```diff
@@ -59,49 +122,102 @@
-{:else}
-	<Button onclick={coinbase}>Buy EOS with Coinbase</Button>
-{/if}
+<h4 class="h4">Token Purchase Providers:</h4>
+
+<MultiCard class="max-w-[700px]">
+	{#each ON_RAMP_PROVIDERS as service}
+		<Card>
+			<div>
+				<div class="mb-4 flex items-center justify-center">
+					<img src={service.logo} alt={service.id} class="h-24 w-3/5 object-contain" />
+				</div>
+				<div class="grid grid-cols-[1fr_auto] text-sm">
+					<div class="flex items-center py-2">
+						<p class="text-md">{m.processing_fees()}</p>
+						<p class="ml-auto text-white">{service.fees.range}</p>
+					</div>
+					<div class="border-gray-300 col-span-2 border-t border-t-[0.5px]"></div>
+					<div class="flex items-center py-2">
+						<p class="text-md">{m.limits()}</p>
+						<p class="ml-auto text-white">{service.limits.daily}</p>
+					</div>
+					<div class="border-gray-300 col-span-2 border-t border-t-[0.5px]"></div>
+				</div>
+				<p class="text-md my-2">{m.details()}</p>
+				<ul class="list-disc pl-6">
+					{#each service.details as detail}
+						<li class="text-sm text-white">{detail}</li>
+					{/each}
+				</ul>
+			</div>
+			<div class="mt-6">
+				{#if !context.account}
+					<p class="text-gray-300 text-sm">{m.must_be_logged_in_for_feature()}</p>
+				{:else}
+					<Button
+						variant="secondary"
+						class="{service.action.className} w-full"
+						onclick={() => handleOnRamp(service.id)}>{service.action.text}</Button
+					>
+				{/if}
+			</div>
+		</Card>
+	{/each}
+</MultiCard>
 
-<h2 class="h2">Exchanges</h2>
-<p>
-	EOS can be purchased through a number of platforms, depending on the users needs and location.
-	Below are some of the most popular options available.
+<h4 class="h4">{m.exchanges()}</h4>
+
+<p class="text-gray-200 mb-6">
+	{m.where_eos_can_be_purchased()}
 </p>
-<ul>
-	<li><a href="https://www.binance.com">https://www.binance.com</a></li>
-	<li><a href="https://www.coinbase.com">https://www.coinbase.com</a></li>
-	<li><a href="https://www.kraken.com">https://www.kraken.com</a></li>
-	<li><a href="https://www.okx.com">https://www.okx.com</a></li>
-	<li><a href="https://global-aws.huobi.com">https://global-aws.huobi.com</a></li>
-	<li><a href="https://www.gate.io">https://www.gate.io</a></li>
-	<li><a href="https://upbit.com">https://upbit.com</a></li>
-	<li><a href="https://kucoin.com">https://kucoin.com</a></li>
-</ul>
+
+<MultiCard class="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-4">
+	{#each EXCHANGES as exchange}
+		<Card class="h-full p-6 pb-2">
+			<a
+				href={exchange.url}
+				target="_blank"
+				rel="noopener noreferrer"
+				class="block h-full transition-all hover:scale-105"
+			>
+				<div class="relative mb-4 aspect-square overflow-hidden rounded-lg bg-mineShaft-900 p-4">
+					<img
+						src={exchange.logo}
+						alt="{exchange.name} logo"
+						class="absolute left-1/2 top-1/2 h-full w-full -translate-x-1/2 -translate-y-1/2 transform object-cover p-4"
+					/>
+				</div>
```

Comment:
All this centering can be replaced with:

```suggestion
				<div class="mb-4 aspect-square grid place-items-center rounded-lg bg-mineShaft-900 p-4">
					<img
						src={exchange.logo}
						alt="{exchange.name} logo"
						class="size-full object-cover p-4"
					/>
				</div>
```

Reply:
Oh awesome!